x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/utils.ts

```diff
@@ -3,13 +3,14 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-
-import { SignalHit } from '../../types';
+import { performance } from 'perf_hooks';
```

Comment:
That's cool, I haven't used this before in NodeJS land. Nice 💎 gem

Reply:
Thanks yeah I was trying to find a way to get more accurate timing and this was referenced in the stack overflow answer lol. New to me too!